src/Components/ProductTable.jsx

```diff
@@ -45,6 +45,26 @@ export default function ProductTable({ cart, updateCart }) {
     setProducts(body);
   }, []);
 
+  useEffect(() => {
+    let sortedProducts = [...products];
+
+    for (let i = 0; i < sortOptions.length; i++) {
+      if (sortOptions[i].current === true) {
+        if (sortOptions[i].name === "Price") {
+          console.log("sorting by price");
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** please remove console.log‏